packages/ses/src/commons.js

```diff
@@ -72,6 +72,8 @@ export const {
   toStringTag: toStringTagSymbol,
   iterator: iteratorSymbol,
   matchAll: matchAllSymbol,
+  keyFor: SymbolKeyFor,
+  for: SymbolFor,
```

Comment:
Elsewhere, as in `arrayPrototype`, there’s precedent for camelCase over PascalCase.

Reply:
Actually thinking about this more, that means we don't differentiate between static and proto methods?